### ISSUE TITLE
Possible to init all fields with a given model ? 

### ISSUE THREAD

**Issue by FranckErnewein** [formsy/formsy-react#64]
Hi,

Instead of setting all fields one by one,
I want to know if it is possible to setup all fields directly by passing a model.

Instead of
```jsx
<Formsy>
  <InputText name="firstname" value={user.firstname} />
  <InputText name="lastname" value={user.lastname} />
</Formsy>
```
Write something like
```jsx
<Formsy model={user}>
  <InputText name="firstname" />
  <InputText name="lastname" />
</Formsy>
```

**Comment by rkuykendall** [comment]
Hey @FranckErnewein! I _almost_ posted that I didn't think this was possible, but just noticed reset takes an object and applies it to all inputs. I think something like this would work:

```js
setRefForm (formComp) {
    this.refForm = formComp;
    this.refForm.reset(user);
}
```

```jsx
<Formsy ref={this.setRefForm}>
  <InputText name="firstname" />
  <InputText name="lastname" />
</Formsy>
```

**Comment by MilosRasic** [comment]
Even though there's a way to achieve this via reset() method, it would be nice to support something like this in the future.

**Comment by rkuykendall** [comment]
This has been fixed in the new 2.0.0 release. The new added function is `formsy.updateInputsWithValue` and it works very similarly to `updateInputsWithError`.

Please re-open this ticket or create a new one if I am mistaken.

# v2.0.0

- Much, much faster
- Many, many bug fixes
- Full Typescript typing
- Fixed deprecated lifecycle methods 
- Updated all libraries
- Fixed a number of legacy decisions in the Formsy API, mostly a reliance on function props over value props passed down to wrapped components. However, the API changes are minor and listed below:
  - `getErrorMessage()` => `errorMessage`
  - `getErrorMessages()` => `errorMessages`
  - `getValue()` => `value`
   `hasValue()` => `hasValue`
  - `isFormDisabled():` => `isFormDisabled`
  - `isFormSubmitted()` => `isFormSubmitted`
  - `isPristine()` => `isPristine`
  - `isRequired()` => `isRequired`
  - `isValid():` => `isValid`
  - `showError()` => `showError`
  - `showRequired()` => `showRequired`

https://github.com/formsy/formsy-react/compare/v1.1.4...v2.0.0
